packages/documentation/src/stories/health/icons/icons.blocks.tsx

```diff
@@ -0,0 +1,104 @@
+// icons.blocks.tsx
+import React from 'react';
+import reportData from '../../../../node_modules/@swisspost/design-system-icons/public/report.json';
+import { IconReport } from './icons-report';
+
+const report = reportData as IconReport;
+
+export const StatusBlock: React.FC = () => (
+  <div className="status">
+    <strong>Version:</strong> {report.version}
+    <br />
+    <strong>Successful:</strong> {report.stats.success}
+    <br />
+    <strong>Errored:</strong> {report.stats.errors}
+    <br />
+    <strong>Not found:</strong> {report.stats.notFound} <br />
```

Comment:
What do you think, can we extend this to show also the amount of icons with a wrong viewboxes, no keywords or no svg?